### PR TITLE
Game corrections and additions

### LINKED ISSUE #442: Add OpenRW
OpenRW

```yaml
- name: Grand Theft Auto III
  names:
    -
  meta:
    genre: [action]
    subgenre: [TPS]
    theme: [criminal]
  remakes:
    - name: OpenRW
      repo: https://github.com/rwengine/openrw
      url: http://openrw.org/
      development: Sporadic
      status: Unplayable
      lang: [English]
      framework: [SDL]
      license: GPLv3
      content:
      info:
      updated:
      images:
        -
      video:
        youtube:
        vimeo:
```

### PR DESCRIPTION
**Pull request by wonea**
Fixes #442 #445 #441 #438 #437 #448 #450

### CODE REVIEW

**Comment by wonea** [comment]
Deployment seems broken

**Comment by cxong** [comment]
`added` is not in the schema; it's `updated`

**Comment by wonea** [comment]
Changed added to updated. Still failing.

**Comment by wonea** [comment]
Any ideas?

**Comment by nikuda** [comment]
Looks like one of the parsed tag fields (lang, license, etc...) is not returning a mappable value. Could be an empty tag field perhaps.

File: games/s.yaml [hunk 1]
```diff
@@ -74,6 +74,21 @@
...
+- name: The Sims Online
+  meta:
+    genre: [MMORPG]
+    subgenre: [Life]
+    theme: [Management]
+  remakes:
+    - name: Project Dollhouse
+      repo: https://github.com/Afr0Games/Project-Dollhouse
+      url: http://www.afr0games.com
+      development:
```
Comment (nikuda) [inline games/s.yaml]: No value set for `development` is what's breaking the build. Remove the key or add a value to it. 😌

**Pull request merged**